blocks-touch/b-slider/b-slider.js

```diff
@@ -127,6 +127,11 @@
             slider._currentX = slider._currentX || 0;
             // объект с текущими параметрами каждого слайда
             slider._touch = {};
+            // страница для отслеживания pointermove / pointerup
+            slider._page = $(window);
```

Comment:
:+1: But I can't find info about it in the docs (http://ru.bem.info/technology/i-bem/v2/i-bem-js/). Am I missed it or this docs doesn't exist?